package.json

```diff
@@ -16,7 +16,8 @@
     "app:edge-dev": "cross-env BROWSER='edge' webpack --config webpack/webpack.dev.js --watch",
     "app:firefox": "rimraf dist && cross-env BROWSER='firefox' webpack --config webpack/webpack.prod.js",
     "app:firefox-dev": "cross-env BROWSER='firefox' webpack --config webpack/webpack.dev.js --watch",
-    "test:unit": "jest"
+    "test:unit": "jest",
+    "test:unit-update": "jest --updateSnapshot"
```

Comment:
what for?

Reply:
can we combine 2 commands into 1?

Reply:
if you run "npm run test:unit-update", does it also run both snapshot & unit test?

Reply:
```suggestion
    "test:unit-update-snapshot": "jest --updateSnapshot"
```